### ISSUE TITLE
creat a demo, will be populate!

### ISSUE THREAD

**Issue by OHeroJ** [vapor-community/styleguide#12]
creat a demo, will be populate!

**Comment by jdmcd** [comment]
There are a few examples out in the wild, most notably https://github.com/vapor-community/repository-template/ and https://github.com/gotranseo/template. Both adhere to the style guide (and will be updated once the final version is merged into the Vapor docs) but the Transeo one is a little more specific to our development environment. If you're just looking to see some of this stuff in practice, check out the repository-template.
